fixtures/test-server-upload-dir-limit/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+setup() {
+  truncate -s 9KB bigfile1 bigfile2 bigfile3
+}
+
+run_test() {
+  result=$(curl -s -F "file=@bigfile1" localhost:8000)
+  result=$(curl -s -F "file=@bigfile2" localhost:8000)
```

Comment:
Can you also check that these two files are uploaded? Adding a single check before attempting to upload `bigfile3` would be enough.